cirq-core/cirq/contrib/requirements.txt

```diff
@@ -11,4 +11,5 @@ autoray
 numba>=0.53.0
 
 # quil import
-pyquil~=2.21.0
+pyquil~=2.28.2; python_version < "3.7"
+pyquil==3.0.0rc22; python_version >= "3.7"
```

Comment:
Do we have to pin to this particular release candidate?  Is a looser pinning possible?  For instance, pyquil>=3.0.0rc22?

Reply:
Yup, I just temporarily had it pinned to the release candidate. We just released pyquil 3.0.0, so I will update it to use pyquil~=3.0.0